.travis.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+  - windows
+language: rust
+rust:
+  - 1.29.2
+sudo: false
+cache:
+  cargo: true
+before_script:
+  - cargo install cargo-prune --force
```

Comment:
These are just going to slow things down. Can you pls use something [like this](https://github.com/ustulation/p2p/blob/example/.travis.yml) instead ?